sdk/keyvault/tests.yml

```diff
@@ -4,6 +4,7 @@ extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
   parameters:
     ServiceDirectory: keyvault
+    SDKType: client
```

Comment:
What SDKType do we use for mgmt?

Reply:
"mgmt"

Reply:
@m-nash do you want me to add live tests for mgmt e.g., a _tests.mgmt.yml_? Doesn't seem typical - just _ci.mgmt.yml_. /cc @benbp

Reply:
No thank you, @ArthurMa1978 can work that out when they move to a process where the live tests are managed / monitored.